python/cudf/cudf/core/multiindex.py

```diff
@@ -645,15 +646,15 @@ def is_unique(self):
     @property
     def is_monotonic_increasing(self):
         if not hasattr(self, "_is_monotonic_increasing"):
-            self._is_monotonic_increasing = self._source_data.argsort(
-                ascending=True
-            ).is_monotonic_increasing
+            self._is_monotonic_increasing = libcudf.issorted.issorted(
+                self._source_data._columns
+            )
         return self._is_monotonic_increasing
 
     @property
     def is_monotonic_decreasing(self):
         if not hasattr(self, "_is_monotonic_decreasing"):
-            self._is_monotonic_decreasing = self._source_data.argsort(
-                ascending=True
-            ).is_monotonic_decreasing
+            self._is_monotonic_decreasing = libcudf.issorted.issorted(
+                self._source_data._columns, [1] * len(self.levels)
+            )
```

Comment:
Can you add a test for multiindexes as well?

Reply:
There is already one in place https://github.com/rapidsai/cudf/blob/64a781cb29a08262e6d72a7fbe847eae21b3473d/python/cudf/cudf/tests/test_monotonic.py#L159